_posts/2024-07-03-oidc-cient-request.adoc

```diff
@@ -0,0 +1,447 @@
+---
+layout: post
+title: 'Sending Request Objects as A JWT Using Request Parameters for OIDC'
+date: 2024-07-03
+tags: oidc Keycloak JWT encryption authentication
+synopsis: How to send the request object as a Json Web Token when securing a WildFly application using OIDC.
+author: prarthonapaul
+---
+
+:toc: macro
+:toc-title:
+
+WildFly 33 includes the ability to send the request object as a Json Web Token (JWT) when securing an application using OIDC.
+The feature also includes the ability to sign and/or encrypt the JWT for added security. https://openid.net/developers/how-connect-works/[OpenID Connect] is an identity layer on top of the OAuth 2.0 protocol
+that makes it possible for a client to verify a user’s identity based on authentication that’s performed by an OpenID
+provider. The OAuth 2.0 request syntax sends the Request Object in the request by adding them directly to the URL.
+However, OpenID Connect allows the Request Object to be sent as a JWT using https://openid.net/specs/openid-connect-core-1_0.html#JWTRequests[request parameters], which can be signed and optionally
+encrypted. This adds an added layer of protection, as the parameters are no longer human-readable.
+
+toc::[]
+
+== Prerequisites 
+
+To follow along with this guide, you will need:
+
+* Roughly 15 minutes
+
+* Access to an OpenShift cluster (try the Red Hat Developer Sandbox for free)
+
+* https://docs.openshift.com/container-platform/4.15/cli_reference/openshift_cli/getting-started-cli.html[OpenShift CLI]
+
+* https://helm.sh/docs/intro/install/[Helm Chart]
+
```

Comment:
I suggest adding a bullet item that provides the link to the test app to this list.